cmd/mdatagen/templates/component_test.go.tmpl

```diff
@@ -53,6 +53,13 @@ func (aneh *assertNoErrorHost) ReportFatalError(err error) {
 	assert.NoError(aneh, err)
 }
 
+func TestREADMEmdPresent(t *testing.T) {
+	require.FileExists(t, "README.md")
+}
```

Comment:
Not sure about this requirement though. I think this is a requirement for our ci/cd not for tests.

Reply:
Not sure how to enforce it otherwise. The ci/cd is using the list of components from otelcontribcol right now to check file presence.

Reply:
Why do we care if other components have a README (that are not in our repos)? In our repos we use ci/cd and can use for example the list from the `versions.yaml` and enforce that each module has a README?

Reply:
I will look at using the list from `versions.yaml`, thanks!